build.xml

```diff
@@ -28,12 +28,12 @@
 
 
 <!-- Where and how to identify installed versions of python -->
-<property name="pythonbin" value="/usr/bin"/>
+<property name="pythonbin" value="./venv/bin"/>
```

Comment:
This configuration will only work with particular ways of installing the Python virtual environment.  I guess this approach is consistent with how CADC is doing python testing with ant.  

TODO: Move to a method that does not have hardcoded paths to system binaries.